README.md

```diff
@@ -71,10 +73,12 @@ New major version indicates that there is an incompatible breaking change.
 
 ### Upgrading CRDs
 
+🛂 **Warning**: Traefik v3 totally removes the crd support for traefik.containo.us CRDs. By default this helm installs the CRDs compatible with v2 also, but Traefik v3 will no longer monitor them. There is no support for deprecation errors, so your existing resources may silently fail to work after upgrade to Traefik v3. [Migration guide from v2 to v3](https://doc.traefik.io/traefik/v3.0/migration/v2-to-v3/)
+
 With Helm v3, CRDs created by this chart can not be updated, cf the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions). Please read carefully release notes of this chart before upgrading CRDs.
 
 ```bash
-kubectl apply --server-side --force-conflicts -k https://github.com/traefik/traefik-helm-chart/traefik/crds/
+kubectl apply --server-side --force-conflicts -f https://github.com/traefik/traefik-helm-chart/traefik/crds/
```

Comment:
It was a "-k" on purpose, to apply `kustomization.yaml`.
Either you can remove `kustomization.yaml` or update it and keep `-k`.

Reply:
Reverted

Reply:
Also forgot the new crd in that file